config/checkstyle/checkstyle.xml

```diff
@@ -41,7 +41,9 @@
 
     <!-- Checks whether files end with a new line.                        -->
     <!-- See http://checkstyle.sf.net/config_misc.html#NewlineAtEndOfFile -->
-    <module name="NewlineAtEndOfFile"/>
+    <module name="NewlineAtEndOfFile">
+		<property name="lineSeparator" value="lf"/>
```

Comment:
Checkstyle was changed exactly because Windows system is in use. Windows system default is "crlf", and all the existing code has "lf"- so without that change checkstyle on Windows fails on every file, expecting "crlf" at the end of file.